dremio_toolkit/env_reader.py

```diff
@@ -19,8 +19,9 @@
 from typing import Dict, Optional
 
 from dremio_toolkit.env_api import EnvApi
-from dremio_toolkit.env_definition import EnvDefinition
+from env_definition import EnvDefinition
```

Comment:
this line causes build errors :) previous absolute import was correct:
```suggestion
from dremio_toolkit.env_definition import EnvDefinition
```